viewdir/layout-template.njk

```diff
@@ -0,0 +1,68 @@
+{% block navbar %}
+
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="shortcut icon" type="image/jpg" href="https://www.kainos.com/favicon.png"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+        <!-- Compiled and minified JavaScript -->
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+    </head>
+    <body>
+        <nav style="background-color: #113e71" class="z-depth-3">
+            <div class="nav-wrapper container">
+                <a href="/index" class="brand-logo"><img src="https://i.ibb.co/y43Xwry/Kainos-Logo-Index.png" height=50px style="margin-top: 4%">
+                    <ul id="nav-mobile" class="right hide-on-med-and-down">
+                        <li>
+                            <a href="http://localhost:6555/">Home</a>
```

Comment:
You cannot guarantee the system will be on `localhost:6555`, but you may be able to guarantee the Home page will be at the root

```suggestion
                            <a href="/">Home</a>
```